### PR TITLE
add node v0.12 to travis

### LINKED ISSUE #414: Add node v0.12 to travis
Node has [released](http://blog.nodejs.org/2015/02/06/node-v0-12-0-stable/) v0.12, this should be included in the travis tests.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/8582913-add-node-v0-12-to-travis?utm_campaign=plugin&utm_content=tracker%2F2451736&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F2451736&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by dduugg**
resolves #414

### CODE REVIEW

**Comment by dduugg** [comment]
Continuation of #415

File: Makefile [hunk 1]
```diff
@@ -7,19 +8,38 @@ MONGO_SETTINGS=MONGO_CONNECTION=${MONGO_CONNECTION} \
...
-	istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -vvv -R tap ${TESTS} && \
-	cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && \
-	rm -rf ./coverage
+coverage:
+	NODE_ENV=test ${MONGO_SETTINGS} \
+	${ISTANBUL} cover ${MOCHA} -- -R tap ${TESTS}
+
+report:
+	test -f ${ANALYZED} && \
+    (npm install coveralls && cat ${ANALYZED} | \
```
Comment (jasoncalabrese) [inline Makefile]: indent doesn't look right here

**Comment by jasoncalabrese** [comment]
some nice cleanup and we learned a few things too :+1:

File: Makefile (hunk 1 above)
Comment (dduugg) [inline Makefile]: good catch, there were mixed spaces & tabs too.

**Comment by jasoncalabrese** [comment]
I'll plan to get this in with your other PR soon

**Pull request merged**